Cql/Cql.Compiler/ILibrarySetExpressionBuilderContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Linq.Expressions;
+using Hl7.Cql.Runtime;
+
+namespace Hl7.Cql.Compiler;
+
+internal interface ILibrarySetExpressionBuilderContext : IBuilderContext
+{
+    DefinitionDictionary<LambdaExpression> LibrarySetDefinitions { get; }
```

Comment:
So are the DefinitionDictionaries in each othe the LibraryExpressionBuilderContexts subsets of this one in LibrarySetExpressionBuilderContexts?

Reply:
I'll add some documentation around this. Yes, the LibrarySetDefinitions is the complete set generated from processing the libraryset. For LibraryDefinitions, this is the both the input and output in the context of a library. The input is created from the library's direct dependencies and used to refer to it in expression refs. It would be wrong to use the libraryset's definitions for this